purrfect-match/src/Api.tsx

```diff
@@ -220,8 +220,22 @@ export class Api {
     }
 
     //should eventually connect up to Api.post
-    public static submitContactInfo = (firstName: string, lastName: string, subject: string, email: string, shelterEmployee: string, message: string): Promise<void> => {
+    public static submitContactInfo = (firstName: string, lastName: string, subject: string, email: string, shelterEmployee: string, message: string): Promise<any> => {
         // Normally, we'd actually submit to the backend and resolve or reject the promise based off of the response (was there an error submitting?)
-        return Promise.resolve();
+        return axios({
+            method: 'post',
+            url: 'http://localhost:8080/contact',
+            data: {
+              firstName: firstName,
+              lastName: lastName,
+              subject: subject,
+              email: email,
+              shelterEmployee: shelterEmployee,
+              message: message
+            }
+          }).catch((e: { message: any; })=>{
+              console.log("Hello");
+              console.log(e.message);
```

Comment:
Remove the catch case or throw the error again at the end, we want the caller of this function to be able to deal with errors.